tests/system/robot/chromeTests.robot

```diff
@@ -53,6 +53,6 @@ ARIA checkbox
 i12147
 	[Documentation]	New focus target should be announced if the triggering element is removed when activated
 	test_i12147
-Table in style display: table
+table using display attrbiute

```

Comment:
There is a typo here

Reply:
By the way, this change is introduced because when the system test fails, a screenshot using that name cannot be generated due to the colon symbol in the name. I'm going to make an issue to address this. I think we should replace symbols in the screenshot name generator that don't match `[A-z0-9 _-]`